locksmith/src/controllers/v2/priceController.ts

```diff
@@ -36,3 +42,56 @@ export const total: RequestHandler = async (request, response) => {
 
   return response.send(charge)
 }
+
+/**
+ * Gets the pricing for the universal card payment method
+ * TODO: support swap and purchase to get the route in USDC if the lock is not USDC
+ * @param request
+ * @param response
+ * @returns
+ */
+export const universalCard: RequestHandler = async (request, response) => {
+  const network = Number(request.params.network)
+  const lockAddress = request.params.lock
+  const { recipients: recipientQ = [], purchaseData = [] } = request.query
+
+  // Setup values
```

Comment:
It is so annoying that we have to do this because `recipientQ` can be a string pr an array...

Is there a way to "force" it to be an array @searchableguy ?

Reply:
You would need type the request type which is also annoying so nope.